msbuild/tests/MarsHabitatPricePredictor/LICENSE.txt

```diff
@@ -0,0 +1,11 @@
+Copyright 2017 Apple Inc.
```

Comment:
@spouliot is this kosher to put in our repo?

Reply:
@jstedfast can we use something custom, that we own ?
the alternative is going thru the oss approval process - and considering it's a test I'm not sure it's worth it

Reply:
done